windowsh.imp

```diff
@@ -0,0 +1,178 @@
+  {"include": ["<poppack.h>", "private", "<Windows.h>", "public"]},
+  {"include": ["<processenv.h>", "private", "<Windows.h>", "public"]},
+  {"include": ["<processthreadsapi.h>", "private", "<Windows.h>", "public"]},
+  {"include": ["<processtopologyapi.h>", "private", "<Windows.h>", "public"]},
+  {"include": ["<profileapi.h>", "private", "<Windows.h>", "public"]},
+  {"include": ["<propidl.h>", "private", "<Windows.h>", "public"]},
+  {"include": ["<propidlbase.h>", "private", "<Windows.h>", "public"]},
+  {"include": ["<pshpack1.h>", "private", "<Windows.h>", "public"]},
+  {"include": ["<pshpack2.h>", "private", "<Windows.h>", "public"]},
+  {"include": ["<pshpack4.h>", "private", "<Windows.h>", "public"]},
+  {"include": ["<pshpack8.h>", "private", "<Windows.h>", "public"]},
+  {"include": ["<realtimeapiset.h>", "private", "<Windows.h>", "public"]},
+  {"include": ["<reason.h>", "private", "<Windows.h>", "public"]},
+  {"include": ["<rpc.h>", "private", "<Windows.h>", "public"]},
+  {"include": ["<rpcasync.h>", "private", "<Windows.h>", "public"]},
+  {"include": ["<rpcdce.h>", "private", "<Windows.h>", "public"]},
+  {"include": ["<rpcdcep.h>", "private", "<Windows.h>", "public"]},
+  {"include": ["<rpcerr.h>", "private", "<Windows.h>", "public"]},
+  {"include": ["<rpcmac.h>", "private", "<Windows.h>", "public"]},
+  {"include": ["<rpcndr.h>", "private", "<Windows.h>", "public"]},
+  {"include": ["<rpcnsi.h>", "private", "<Windows.h>", "public"]},
+  {"include": ["<rpcnsip.h>", "private", "<Windows.h>", "public"]},
+  {"include": ["<rpcnterr.h>", "private", "<Windows.h>", "public"]},
+  {"include": ["<rpcsal.h>", "private", "<Windows.h>", "public"]},
+  {"include": ["<sal.h>", "private", "<Windows.h>", "public"]},
+  {"include": ["<sdkddkver.h>", "private", "<Windows.h>", "public"]},
+  {"include": ["<securityappcontainer.h>", "private", "<Windows.h>", "public"]},
+  {"include": ["<securitybaseapi.h>", "private", "<Windows.h>", "public"]},
+  {"include": ["<setjmp.h>", "private", "<Windows.h>", "public"]},
+  {"include": ["<setjmpex.h>", "private", "<Windows.h>", "public"]},
+  {"include": ["<smmintrin.h>", "private", "<Windows.h>", "public"]},
+  {"include": ["<softintrin.h>", "private", "<Windows.h>", "public"]},
+  {"include": ["<specstrings.h>", "private", "<Windows.h>", "public"]},
+  {"include": ["<specstrings_strict.h>", "private", "<Windows.h>", "public"]},
+  {"include": ["<specstrings_undef.h>", "private", "<Windows.h>", "public"]},
+  {"include": ["<stdarg.h>", "private", "<Windows.h>", "public"]},
+  {"include": ["<stddef.h>", "private", "<Windows.h>", "public"]},
+  {"include": ["<stdint.h>", "private", "<Windows.h>", "public"]},
+  {"include": ["<stdio.h>", "private", "<Windows.h>", "public"]},
+  {"include": ["<stdlib.h>", "private", "<Windows.h>", "public"]},
+  {"include": ["<stralign.h>", "private", "<Windows.h>", "public"]},
+  {"include": ["<string.h>", "private", "<Windows.h>", "public"]},
```

Comment:
I think it is wrong to mark C standard library headers as private and to map them to `<Windows.h>`.